bluebutton-data-model-rif/src/main/resources/db/migration/V15__Add_Inner_Join_Relationships.sql

```diff
@@ -0,0 +1,21 @@
+/*
+ * Alters the Beneficiaries History table to add foreign key to Beneficiary 
+ * 
+ * See:
+ * * https://jira.cms.gov/browse/BLUEBUTTON-620
+ */
+
+alter table "BeneficiariesHistory" add column "beneHistoryParentBeneficiary" varchar(15);
```

Comment:
Oops, missed this earlier. Doesn't this table already have a `beneficiaryId` column? Can you get JPA to re-use that? If not, you're going to have to populate this column yourself.

Reply:
After poking around some more, I'm not sure that I can reuse the `beneficiaryId` column like that. Since it is just a string, and not an actual reference to the `Beneficiary` entity. Or at least, I can't figure out how to get that to work... Did you have something specific in mind? Otherwise I think populating the column is going to be the way to go.

Reply:
What's the failure you get when you try? This _should_ be possible, and I hate adding columns just to accommodate our ORM.

Feel free to throw a 30 minute pairing onto my calendar for tomorrow after 3pm.